api/src/main/resources/liquibase.xml

```diff
@@ -306,4 +306,33 @@
             <column name="uuid" value="8b5caeea-2c3b-11e8-b467-0ed5f89f718b" />
         </insert>
     </changeSet>
+
+	<changeSet author="rmakoba" id="ugandaemr-04052020-iss-9">
+		<preConditions onFail="MARK_RAN">
+			<not><tableExists tableName="public_holiday"/></not>
+		</preConditions>
+		<comment>
+			Create the public_holiday table
+		</comment>
+        <createTable tableName="public_holiday">
+            <column autoIncrement="true" name="public_holiday_id" type="INT">
+                <constraints primaryKey="true"/>
+            </column>
+            <column  name="public_holiday_date" type="datetime">
```

Comment:
@reagan-meant the public_holiday_date field needs to be a date not datetime, and add a unique constraint on it so that no duplicates can be added

Reply:
Dont